### PR TITLE
Mes 4292 comms page flow cat d

### PR DESCRIPTION
**Pull request by a0dexx**
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [x] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Comment by a0dexx** [comment]
this is adding CAT D to the Comms page

**Review by edwardwilson — APPROVED** [review]

File: src/pages/communication-page/cat-d/communication.cat-d.page.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<ion-header>
+  <ion-navbar>
+    <ion-title>{{ 'communication.title' | translate }} - {{pageState.candidateUntitledName$ | async}}</ion-title>
+    <ion-buttons end>
+      <end-test-link category="C"></end-test-link>
```
Comment (TomBillingtonUK) [inline src/pages/communication-page/cat-d/communication.cat-d.page.html]: we should get the category from state here and pass it in

**Review by TomBillingtonUK — APPROVED** [review]

File: src/pages/communication-page/cat-d/communication.cat-d.page.html (hunk 1 above)
Comment (a0dexx) [reply src/pages/communication-page/cat-d/communication.cat-d.page.html]: done

**Comment by a0dexx** [comment]
https://jira.dvsacloud.uk/browse/MES-4292

**Comment by a0dexx** [comment]
https://jira.dvsacloud.uk/browse/MES-4292

**Pull request merged**